packages/editor/src/components/post-featured-image/index.js

```diff
@@ -54,21 +69,24 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 						allowedTypes={ ALLOWED_MEDIA_TYPES }
 						modalClass={ ! featuredImageId ? 'editor-post-featured-image__media-modal' : 'editor-post-featured-image__media-modal' }
 						render={ ( { open } ) => (
-							<Button
-								className={ ! featuredImageId ? 'editor-post-featured-image__toggle' : 'editor-post-featured-image__preview' }
-								onClick={ open }
-								aria-label={ ! featuredImageId ? null : __( 'Edit or update the image' ) }>
-								{ !! featuredImageId && media &&
-									<ResponsiveWrapper
-										naturalWidth={ mediaWidth }
-										naturalHeight={ mediaHeight }
-									>
-										<img src={ mediaSourceUrl } alt="" />
-									</ResponsiveWrapper>
-								}
-								{ !! featuredImageId && ! media && <Spinner /> }
-								{ ! featuredImageId && ( postLabel.set_featured_image || DEFAULT_SET_FEATURE_IMAGE_LABEL ) }
-							</Button>
+							<>
+								<DropZone onFilesDrop={ ! featuredImageId && onDropImage } />
```

Comment:
There's a message that is shown in the dropzone while hovering it, it's not aligned properly I think.

Reply:
It was the order: https://github.com/WordPress/gutenberg/pull/17486/commits/cdd7d6614a493887f41a37269dd6fd8d93c0615e